api/migrations/versions/0199_change_provider_details.py

```diff
@@ -57,7 +57,7 @@ def make_provider_details_downgrade_sql(identifier):
             ORDER BY version desc
             LIMIT 1
         ) h
-        WHERE identifier = '{0}'
+        WHERE provider_details.identifier = '{0}'
```

Comment:
@andrewoh531 is this required?

Reply:
sql complaints that it is ambiguous otherwise

Reply:
Cool, thanks for fixing my rubbish SQL.

Reply:
Yeah it was complaining of ambiguous column name (line number in the logs doesn't match up to the file):
```
sqlalchemy.exc.ProgrammingError: (psycopg2.ProgrammingError) column reference "identifier" is ambiguous
LINE 19:         WHERE identifier = 'firetext'
```